ansible/roles/riakcs/configure/templates/advanced.config.j2

```diff
@@ -0,0 +1,16 @@
+[
+    {riak_kv, [
+        {add_paths, ["{{ riakcs_ebin }}"]},
+        {storage_backend, riak_cs_kv_multi_backend},
+        {multi_backend_prefix_list, [{<<"0b:">>, be_blocks}]},
+        {multi_backend_default, be_default},
+        {multi_backend, [
+            {be_default, riak_kv_eleveldb_backend, [
+                {data_root, "{{ riak_data_root_leveldb }}"}
+            ]},
+            {be_blocks, riak_kv_eleveldb_backend, [
```

Comment:
On the old nodes this uses bitcask but daniel said it's ok that these will be using this